lmcache/v1/token_database.py

```diff
@@ -136,7 +139,7 @@ def _make_key_by_hash(self, chunk_hash: int):
         )
 
     def _get_init_hash(self) -> int:
-        return NONE_HASH
+        return 0  # Override vLLM's random NONE_HASH for cross-process cache sharing.
```

Comment:
there is in the `_hash` function below `hash_func((prefix_hash, tokens_tuple, None))`

and if prefix_hash is not None, but 0 at the start, it does not help with the third tuple object